.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Release
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
why is this set to run on pushes to main? We don't want to release on every push to main, right?

Reply:
Thanks for the explanation, sounds good to me